ext/nnstreamer/tensor_filter/tensor_filter_tvm.cc

```diff
@@ -219,7 +219,7 @@ tvm_subplugin::configure_instance (const GstTensorFilterProperties *prop)
 
   for (unsigned int i = 0; i < inputInfo.num_tensors; ++i) {
     arr = getInput (i);
-    dt = arr.operator-> ();
+    dt = const_cast<DLTensor*> (arr.operator-> ());
```

Comment:
I wonder the reasoning behind this change.

Reply:
Apparently there's no need for this..